e2e/pvc_check.go

```diff
@@ -11,9 +12,23 @@ import (
 	"k8s.io/apimachinery/pkg/types"
 )
 
-const (
-	size = "5Gi"
-)
+//go:embed testdata/pvc_tests/pvc-template.yaml
+var pvcYAMLTemplate string
```

Comment:
`YAML` can be removed from each of the variables. Looks redundant IMO.  `pvcTemplate` should suffice.

Reply:
I think the yaml makes it a little clearer as to what type it is in the code.

Reply:
ok